automation/terraform/testnets/finalfinal2/finalfinal3_seeds.txt

```diff
@@ -0,0 +1,27 @@
+/dns4/seed-1.finalfinal2.o1test.net/tcp/10000/p2p/12D3KooWGb3tna8uSs1dyTVX5qMRbxQzTXmyGJMLw2hFZePCzSTg
```

Comment:
what is finalfinal2_seeds vs finalfinal3_seeds? (what is finalfinal3?)

Reply:
finalfinal3_seeds is the same file, there was a bad peer string in the first version I uploaded to google for finalfinal2 and for a few hours GCP was caching it, so i made finalfinal3 to avoid the cache hit and get people moving quickly. It shouldn't happen again if we properly verify community peer ID's before uploading